styles/Plans.module.css

```
@@ -0,0 +1,15 @@
+.container {
+    @apply max-w-[1200px] mx-auto flex flex-col justify-center h-full items-center;
+}
+
+.containerPlans {
+    @apply grid grid-cols-1 lg:grid-cols-3 gap-1;
+}
+
+.descriptionPlans {
+    @apply shadow-lg rounded-lg py-10 px-12 bg-[#2f6937] mr-3 ml-3 text-white;
+}
+
+.btn {
+    @apply flex font-medium rounded-lg text-sm px-10 py-2.5 mr-2;
+}
```

Add end of file line
```suggestion
}

```